stingray/lightcurve.py

```diff
@@ -29,8 +33,16 @@ def __init__(self, time, counts, input_counts=True, gti=None):
 
         counts: iterable, optional, default None
             A list or array of the counts in each bin corresponding to the
-            bins defined in `time` (note: **not** the count rate, i.e.
-            counts/second, but the counts/bin).
+            bins defined in `time` (note: use `input_counts=False` to
+            input the count rage, i.e. counts/second, otherwise use
+            counts/bin).
+
+        err: iterable, optional, default None:
+            A list or array of the uncertainties in each bin corresponding to
+            the bins defined in `time` (note: use `input_counts=False` to
+            input the count rage, i.e. counts/second, otherwise use
+            counts/bin). If None, we assume the data is poisson distributed
+            and calculate the error as `1 + sqrt(N + 0.75)`
```

Comment:
Gehrels something. However, this is not what we use anymore, we use the average of the lower and upper confidence intervals